stylesheets/components/avatar.css

```diff
@@ -0,0 +1,74 @@
+@import url("/stylesheets/main.css");
+
+.avatar {
+  margin: 1rem;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 100%;
+  position: relative;
+}
+
+.avatar-sm {
+  width: 2rem;
+  height: 2rem;
+}
+
+.avatar-lg {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+.avatar-xl {
+  width:4rem;
+  height:4rem;
+}
+
+.avatar-sq {
+  border-radius: 0.2rem;
+}
+
+.avatar-container {
+  position: relative;
+  width: fit-content;
+}
+
+.badge-online {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color: var(--badge-online);
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
+  border: 1.5px solid white;
+}
+
+.badge-idle {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color: var(--badge-idle);
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
+  border: 1.5px solid white;
+}
+
+.badge-dnd {
+  width: 15px;
+  height: 15px;
+  border-radius: 100%;
+  background-color:var(--badge-dnd) ;
+  position: absolute;
+  bottom: 25px;
+  left: 50px;
+  border: 1.5px solid white;
+}
+
+.av-cont {
```

Comment:
Just a suggestion: you can attach this piece of code as an utility class in your stylesheets folder that has the property **display:flex** because this one in particular might be used frequently!!
Rest of the code looks good to me👍🏻 
had a small clarification : in the badge-online, badge-idle classes for properties top, left, bottom,right we should give the values in absolute units px or does using rems also works?

Reply:
Thanks for suggestion ,will implement it.
Also, for using px or rem regarding position property, in my opinion rem should also work fine.

Reply:
It is suggested to use percentage rather than hard coded values in px or rem for positioning an element, because if the parent's dimension changes, the position of the element would remain the same if you are using px or rem, but suppose you ar using `bottom:0%; right:0%`, it would always remain at the bottom right of the parent